### DISCUSSION TITLE
Question about state of charge

### DISCUSSION THREAD

**Discussion by YouHyin** [pybamm-team/PyBaMM#2620] [Q&A]
Hello, I am currently conducting a charging protocol optimization study using Pybamm.
 I want to proceed with the code like "Charge until 98% SOC", is this function possible in electrode SOH?
https://github.com/pybamm-team/PyBaMM/issues/2553

I'd like to designate a SOC during the charging process, but I'm really curious if it's possible. I would appreciate it if you could reply.

**Comment by L4XIA** [comment]
Not possible right now, PyBaMM Team thinks there is an ambiguity in defining the SOC. But you may achieve 'Charge until 98% SOC' by using re-initialized solutions with experiment or using solution steps if cycling without events.

**Reply by YouHyin** [reply]
I don't understand "re-initialized solutions with experiment".
Could you tell me how to do this?
